ya_httpx_client/session.py

```diff
@@ -115,6 +21,7 @@ def __init__(self, request_queue):
         self.request_queue = request_queue
 
     async def handle_async_request(self, method, url, headers, stream, extensions):
+        # pylint: disable=too-many-arguments
```

Comment:
In `.pylintrc`:
```
[DESIGN]
max-args=6
```

Reply:
As discussed, I left it without changes

Reply:
On second thoughts, better leave it as it is (a local exception to the rule).